{
    "title" : "FailedToImpersonateUsersException after upgrading to 7.0",
    "uri" : "http://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "printableUri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "clickUri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "excerpt" : "This question was closed Aug 08, 2014 at 04:29 PM by Carl Bolduc. ... certificateimpersonateimpersonation ... FailedToImpersonateUsersException after upgrading to 7.0 ... I just upgrade coveo on ou...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1030,
    "percentScore" : 72.21627,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "FailedToImpersonateUsersException after upgrading to 7.0",
      "sysauthor" : "mmintz",
      "sysurihash" : "2tgyEBYQhqmFezcn",
      "urihash" : "2tgyEBYQhqmFezcn",
      "numberofanswers" : 0,
      "sysuri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160347,
      "questionaskeddatestring" : "·\n            Aug 08, 2014 at 10:55 AM ·",
      "tags" : [ "certificate", "impersonate", "impersonation" ],
      "sysconcepts" : "certification ; coveo ; upgrade ; FailedToImpersonateUsersException ; configuration setting ; web configs ; Carl Bolduc ; access the search",
      "concepts" : "certification ; coveo ; upgrade ; FailedToImpersonateUsersException ; configuration setting ; web configs ; Carl Bolduc ; access the search",
      "printableuri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517941824000,
      "permanentid" : "1be99c0f6d243d0f4f0adc41b5bff8c076265c77a56385b644d599f0d9cf",
      "syslanguage" : [ "English" ],
      "transactionid" : 160347,
      "title" : "FailedToImpersonateUsersException after upgrading to 7.0",
      "date" : 1407456000000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "hasanswer" : "false",
      "audience" : [ "Administrator", "Developer" ],
      "commentauthor" : [ "Carl Bolduc" ],
      "hasacceptedanswer" : "false",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5447036,
      "numberofcomments" : 1,
      "size" : 65432,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911577479,
      "acceptedanswers" : 0,
      "syssize" : 65432,
      "sysdate" : 1407456000000,
      "author" : "mmintz",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>I just upgrade coveo on our production site from 6.5 to 7.0. When I try to access the search page on the site, I receive \"FailedToImpersonateUsersException\" even though the certification was created during setup. I check the web configs and all the references to the certificate are there and pointed to the correct location. </p> \n<p>Is there a configuration setting that I'm missing somewhere? </p>\n                </div>",
      "sitemaplastmodified" : 1407456000000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517941824000,
      "questionnumber" : 2237,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "mmintz",
      "sitename" : "Answers",
      "sysrowid" : 5447036,
      "uri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "FailedToImpersonateUsersException after upgrading to 7.0",
    "Uri" : "http://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "PrintableUri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "ClickUri" : "https://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/2237/failedtoimpersonateusersexception-after-upgrading.html",
    "Excerpt" : "This question was closed Aug 08, 2014 at 04:29 PM by Carl Bolduc. ... certificateimpersonateimpersonation ... FailedToImpersonateUsersException after upgrading to 7.0 ... I just upgrade coveo on ou...",
    "FirstSentences" : null
  }